docs/docs/native-packages.md

```diff
@@ -0,0 +1,65 @@
+# Native Packages
```

Comment:
Is this page automatically picked up by our docs?

Reply:
It's linked in index.md